purchases/src/customEntitlementComputation/kotlin/com/revenuecat/purchases/Purchases.kt

```diff
@@ -181,6 +181,11 @@ class Purchases internal constructor(
     }
     //endregion
 
+    @ExperimentalPreviewRevenueCatPurchasesAPI
+    sealed class DeepLink {
+        class WebRedemptionLink internal constructor(internal val redemptionToken: String) : DeepLink()
```

Comment:
Made everything internal. Devs shouldn't need to build an instance of this on their own, nor access the token directly.

Reply:
Good call! What's the reason this is nested in `Purchases`? (Don't really have an opinion, just curious haha.)

Reply:
Mostly seemed `DeepLink` was too generic as a top-level class 😅 So I wanted to namespace it inside the SDK.

Reply:
Haha alright makes sense! (Devs can always use import aliasing of course.)